make/prepare

```diff
@@ -45,10 +45,11 @@ secret_dir=${data_path}/secret
 config_dir=$harbor_prepare_path/common/config
 
 # Run prepare script
-docker run --rm -v $input_dir:/input:z \
-                    -v $harbor_prepare_path:/compose_location:z \
-                    -v $config_dir:/config:z \
```

Comment:
The removal of this `:z` will break the ability to install harbor if you're running with selinux enabled.
 #8129

Reply:
You are right. Thanks for mention